Functions/xASL_io_ReadDataPar.m

```diff
@@ -166,7 +171,9 @@
 	x = xASL_io_ReadDataPar_FixFields(x);
 	
 	%% Check deprecated fields
-    x = xASL_io_CheckDeprecatedFieldsX(x,true);
+    if bDataPar
```

Comment:
Seems as if this was done by default previously, is this needed for the studypar as well? Or did it previously just do nothing inside this function?

Reply:
Do you remember the studyPar.json that Beatriz showed us yesterday? It had some of these basic fields that shouldn't be there and that were in a sub-structure. That's partially caused by `xASL_io_CheckDeprecatedFieldsX`, which was only recently added to `xASL_io_ReadDataPar` (#717).